stable/datadog/templates/daemonset.yaml

```diff
@@ -274,6 +284,16 @@ spec:
             path: {{ default "/var/lib/datadog-agent/logs" .Values.datadog.logsPointerHostPath | quote }}
           name: pointerdir
         {{- end }}
+        {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
```

Comment:
Why do we need to add `logsConfigContainerCollectAll` flag, what happens if the flag is not set and the user defines a logs autodiscovery annotation ?